cEP-0005.md

```diff
@@ -225,7 +251,7 @@ This allows to show more information in results. Bears can give a custom result
 message; however, by default the message can be generated from the settings of
 the aspects with the ``result_message`` function.
 
-Aspects can also define a ``style_paragraph`` function that returns a small
```

Comment:
undo this change.  It doesnt belong as part of your commit.
I've no objection to a new PR which alters the structure of this sentence such that the word aspects is not at the beginning of the sentence.